.github/workflows/pr_ci.yml

```diff
@@ -29,20 +29,6 @@ jobs:
     - name: Unit Tests
       run: |
         dotnet test -c Release --no-restore --no-build ./test/OrchardCore.Tests/OrchardCore.Tests.csproj
-    - name: Functional Tests
-      if: matrix.os == 'ubuntu-latest'
-      run: |
-        cd test/OrchardCore.Tests.Functional
-        npm install
-        npm run cms:test
-        npm run mvc:test
-    - uses: actions/upload-artifact@v2
-      if: matrix.os == 'ubuntu-latest' && failure()
-      with:
-        name: functional-test-failure
-        path: |
-          test/OrchardCore.Tests.Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data/logs
```

Comment:
Do you want to remove the Cypress functional tests?

Reply:
Eventually, yes, this is a direct substitute.